Tasks:Functions.js

```diff
@@ -11,6 +11,8 @@ function min(x, y) {
 function min(x, y) {
     return x < y ? x : y;
 }
+// Вариант №3
+var min = (x, y) => {return x < y ? x : y; }
```

Comment:
Ты когда пишешь однострочную функцию с оператором return, тебе не нужны фигурные скобки и слово return совсем. Это правило
```suggestion
var min = (x, y) => x < y ? x : y;
```